modelator/_version.py

```diff
@@ -0,0 +1,3 @@
+import importlib.metadata
+
+__version__ = importlib.metadata.version("modelator")
```

Comment:
I explained in my Atomkraft PR why this may not be a good idea.

Anyway, I used a separate file in my PR, because it contains a single line and it is easy to modify when I prepare a release.

If you choose to use `importlib.metadata`, you don't need to modify anything here. So you may put this directly into `__init__.py`. :slightly_smiling_face:

Reply:
Agreed. I will not use this library, also to keep it the same as atomkraft.